cli_tools/common/utils/logging/service/daisy_loggable.go

```diff
@@ -15,13 +15,17 @@
 package service
 
 import (
-	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 	"strconv"
 	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 )
 
-// WorkflowToLoggable provides a Loggable from a daisy workflow.
-func WorkflowToLoggable(wf *daisy.Workflow) Loggable {
+// NewWorkflowToLoggable provides a Loggable from a daisy workflow.
+func NewWorkflowToLoggable(wf *daisy.Workflow) Loggable {
```

Comment:
I'm a bit confused by the name of this func: NewWorkflowToLoggable sounds like a new workflow is created? Or a loggable is created from a new workflow?

Maybe call it NewLoggable or similar?

Reply:
Good point: `NewWorkflowToLoggable` is a funky name. Switched to `NewLoggableFromWorkflow`